nixos/tests/nextcloud/with-declarative-redis-and-secrets.nix

```diff
@@ -62,6 +65,7 @@ in {
 
       services.postgresql = {
         enable = true;
+        package = pkgs.postgresql_14;
```

Comment:
Why is this needed?

Reply:
otherwise the setup fails:

```
nextcloud # [    8.120489] systemd[1]: Started PostgreSQL Server.
nextcloud # [    8.124548] systemd[1]: Starting nextcloud-setup.service...
nextcloud # [    9.224207] postgres[999]: [999] WARNING:  no privileges could be revoked for "nextcloud"
nextcloud # [    9.302305] postgres[992]: [992] ERROR:  permission denied for schema public at character 14
nextcloud # [    9.303310] postgres[992]: [992] STATEMENT:  CREATE TABLE oc_migrations (app VARCHAR(255) NOT NULL, version VARCHAR(255) NOT NULL, PRIMARY KEY(app, version))
nextcloud # [    9.312801] nextcloud-setup-start[982]: Error while trying to initialise the database: An exception occurred while executing a query: SQLSTATE[42501]: Insufficient privilege: 7 ERROR:  permission denied for schema public
nextcloud # [    9.314932] nextcloud-setup-start[982]: LINE 1: CREATE TABLE oc_migrations (app VARCHAR(255) NOT NULL, versi...
nextcloud # [    9.316465] nextcloud-setup-start[982]:                      ^
nextcloud # [    9.319421] nextcloud-setup-start[982]: Trace: #0 /nix/store/ay71npxcw8gafabr4vaxsn6pkbdm5xmc-nextcloud-27.1.4/3rdparty/doctrine/dbal/src/Connection.php(1814): Doctrine\DBAL\Driver\API\PostgreSQL\ExceptionConverter->convert(Object(Doctrine\DBAL\Driver\PDO\Exception), Object(Doctrine\DBAL\Query))
```